system/ee/installer/updates/ud_6_00_04.php

```diff
@@ -48,7 +48,7 @@ private function addreCaptchaAction()
         ee()->db->insert('actions', array(
             'class' => 'Member',
             'method' => 'recaptcha_check',
-            'csrf_exempt' => 0
+            'csrf_exempt' => 1
```

Comment:
If we're making this `csrf_exempt` we also need to make sure it's still excempt when updating from `6.0.6` to `6.1.0`, so need to perform change in latest `ud_` file as well

Reply:
@intoeetive Per conversation, moving this to 6.1 check, and reverting this change.